charts/keycloakx/ci/h2-values.yaml

```diff
@@ -4,47 +4,29 @@ command:
   - "start-dev"
 
 extraEnv: |
-  - name: KEYCLOAK_ADMIN
-    value: "admin"
-  - name: KEYCLOAK_ADMIN_PASSWORD
-    value: "secret"
-  - name: KC_DB
-    value: dev-file
   - name: KC_METRICS_ENABLED
     value: "true"
   - name: KC_HTTP_RELATIVE_PATH
     value: "/auth"
-  - name: KC_DB_USERNAME_FILE
-    value: /secrets/admin-creds/user
-  - name: KC_DB_PASSWORD_FILE
-    value: /secrets/admin-creds/password
   - name: JAVA_OPTS_APPEND
     value: >-
       -XX:+UseContainerSupport
       -XX:MaxRAMPercentage=50.0
       -Djava.net.preferIPv4Stack=true
       -Djava.awt.headless=true
 
+database:
+  vendor: dev-file
+  username: admin
+  password: secret
+
 secrets:
   admin-creds:
     annotations:
       my-test-annotation: Test secret for {{ include "keycloak.fullname" . }}
     stringData:
-      user: admin
-      password: secret
-
-extraVolumeMounts: |
-  - name: admin-creds
-    mountPath: /secrets/admin-creds
-    readOnly: true
-
-extraVolumes: |
-  - name: admin-creds
-    secret:
-      secretName: '{{ include "keycloak.fullname" . }}-admin-creds'
-
-postgresql:
-  enabled: false
+      user: keycloakadmin
```

Comment:
I think this should be just `admin` and `secret` to match with the values for the login test